package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "nyc mocha src/tests/*.js",
```

Comment:
We'll be using ES6 features, can you add babel? Check #6

Reply:
> We'll be using ES6 features, can you add babel? Check #6

Sure. Let me work on that.

Reply:
I have added babel.

Reply:
> We'll be using ES6 features, can you add babel? Check #6

I have added babel.